processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -519,11 +532,27 @@ public AddToFactsResult(
       this.parseExceptionMessages = parseExceptionMessages;
     }
 
+    public AddToFactsResult(
```

Comment:
I'd suggest getting rid of the other constructor. Having too many legacy constructors makes it easy to accidentally call the wrong constructor and leave out an important parameter.

IMO - in this case, if addToFacts isn't able to generate a bytesInMemory number, it would be better to fess up and explicitly pass in a zero so it's obvious to readers what is going on.

Reply:
Removed the other constructor and passed `0` for `bytesInMemory` from `OffheapIncrementalIndex`.